torch/lib/c10d/ProcessGroupNCCL.cpp

```diff
@@ -384,7 +384,7 @@ void ProcessGroupNCCL::WorkNCCL::synchronizeInternal(
       // Check for errors and throw appropriate exception.
       checkAndThrowException();
       std::this_thread::sleep_for(
-          std::chrono::milliseconds(kSynchronizeBusyWaitMillis));
+          std::chrono::microseconds(kSynchronizeBusyWaitMicros));
```

Comment:
We hope to get rid of this busy wait in https://github.com/pytorch/pytorch/pull/45236, but I guess its fine to land this change if it is something urgent for classy vision.

Reply:
Sorry, I was not aware this file is pulled into this PR. I will remove it.

Reply:
No worries, although I was wondering if we were planning on making these changes as part of a separate PR?